lib/resourcemerge/appt_test.go

```diff
@@ -0,0 +1,113 @@
+package resourcemerge
```

Comment:
typo in the file name, should be apps_test.go

Reply:
Whoops, nice catch, I've fixed it, thanks!